test/runtime/fake_vqeruntime.py

```diff
@@ -0,0 +1,99 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Fake runtime provider and VQE runtime."""
+
+from typing import Dict, Any
+import numpy as np
+from qiskit.circuit import QuantumCircuit
+from qiskit.opflow import PauliSumOp
+from qiskit.providers import Provider
+from qiskit_nature.runtime import VQEProgramResult
+
+
+class FakeVQEJob:
```

Comment:
@Cryoris maybe instead of `Fake` we can use `Test`?  It just seems a bit strange to me. But this is only minor.

Reply:
ok I see that by Fake you mean something that mimics the VQEJob. SHould be fine then.

Reply:
I was stealing the name from `FakeBackend` 😄 We could call it `Mock` if you prefer, but I think `Test` is only for classes that test something, whereas this is a "mock" or "fake" class that pretends to be a job.

Reply:
I think is fine. I misinterpret it in the first pass, that's why I marked it as resolved ;)